### PR TITLE
ledger: Enable mainnet.

### LINKED ISSUE #3865: multi: Add Ledger support.
As approved in the [proposal](https://proposals.decred.org/record/609db9e) we are adding ledger support to decrediton. This issue is intended to track those changes.

The goal is to have the ability to sign and send transactions, receive transactions, view transaction history and verify addresses associated with the Ledger in a *new* scene based heavily off of the current trezor flow.

Changes will be made in small increments with as many tests as possible. I would like to disable mainnet use until everything is well tested and refined.

One thing I am not sure of is a small ledger icon we will need to use. If anyone reading this can produce one without much effort please let me know.

TODO:
- [x] add ledger backend methods
- [x] remove tickets section from trezor and add warning to obtaining address screen about only using in p2pkh scripts.
- [ ] add new screens for ledger and hook those up to the ledger methods
- [x] create the small ledger icon on the create/choose wallet page
- [ ] finalize (testing/debugging) and allow on mainnet
- [ ] make tests for LedgerActions.js methods using a device and the testnet decred app (time/funding permitting)

<details closed>
<summary>change log</summary>

```
2023-04-28 Add TODO checklist
2023-09-21 Check add ledger backend methods, create small ledger icon, and remove tickets section
```
</details>

### PR DESCRIPTION
**Pull request by JoeGruffins**
Depends on #3874
closes #3865

### CODE REVIEW

**Comment by JoeGruffins** [comment]
@matheusd let me know if we need anything else to enable here, besides testing this

I am testing on mainnet on linux and windows with nano x and nano s plus. I believe @amass01 is testing on mac? Thanks all for your support almost done, hopefully.

**Review by matheusd — APPROVED** [review]
Tested on mainnet, on both Linux and Windows with a Nano-S. I also tested recovery using software only, by using my bip39-to-dcr-seed tool. Nice work!

**Comment by amass01** [comment]
dále, i will test on m1 mac today 🙏🏽

File: app/actions/LedgerActions.js [hunk 1]
```diff
@@ -155,11 +158,24 @@ function doWithTransport(fn) {
       return fn(transport).then((r) =>
         transport
           .close()
-          .catch((/*e*/) => {}) // throw?
+          .catch((e) => {
+            throw e;
+          })
```
Comment (amass01) [inline app/actions/LedgerActions.js:163]: redundant, it will be thrown by default

Comment (JoeGruffins) [reply app/actions/LedgerActions.js:163]: Ah, will keep this in mind and fix at some point.

Comment (JoeGruffins) [reply app/actions/LedgerActions.js:163]: Was mac alright?

**Pull request merged**